.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+<!-- This is a comment. You can remove it and other comments while filling out the template -->
+
+# Summary
+
+<!-- 
+Please provide a **short** summary in a few sentences of what the PR does.
+Try to keep it to as few full sentences as possible.
+-->
+
+## Checklist
+
+<!--
+Check off and cross out points that don't apply:
+
+Example
+
+ - [x] ~~Marked as draft PR while still working on PR~~ (ready from get-go)
+-->
+
+ - [ ] Marked as draft PR while still working on PR
+ - [ ] Marked as "ready for review" once not in progress
+ - [ ] Filled in all sections
+ - [ ] Added tests (for code changes)
+ - [ ] Provided screenshots (for visual changes)
+
+
+# Additional information
+
+<!--
+In this section you can describe more in depth:
+
+ - **what** your PR does
```

Comment:
Why your PR does should be already stated in the "Summary". 

So this additional information section should only be needed for additional design choices, screenshots, or anything else that helps evaluating this PR.

Reply:
Good point. Removed that line.